src/test/features/eligibility/routes/claim-is-for-tenancy-deposit.ts

```diff
@@ -21,42 +20,40 @@ describe('Claim eligibility: is claim for tenancy deposit page', () => {
   attachDefaultHooks(app)
 
   describe('on GET', () => {
+    checkAuthorizationMiddleware(app, 'get', pagePath)
+
     it('should render page when everything is fine', async () => {
 
       await request(app)
         .get(pagePath)
-        .set('Cookie', `${cookieName}=ABC`)
         .expect(res => expect(res).to.be.successful.withText(expectedTextOnPage))
     })
   })
 
   describe('on POST', () => {
-    describe('for authorized user', () => {
```

Comment:
Despite bigger diff not many changes there, I just removed `describe('for authorized user', () => {` wrapper after making eligibility pages not use session cookie by default.